libCacheSim/include/libCacheSim/cacheObj.h

```diff
@@ -139,6 +139,7 @@ struct cache_obj;
 typedef struct cache_obj {
   struct cache_obj *hash_next;
   obj_id_t obj_id;
+  uint64_t fingerprint;
```

Comment:
Can we use less space for fingerprint? like one-byte? or guard with a macro? Because we may have billions of cache_obj in simulations, we are very sensitive to the memory usage.

Reply:
you can leave it if you do not plan to merge into develop branch